### ISSUE TITLE
Way/area selection impossible

### ISSUE THREAD

**Issue by homersimpsons** [openstreetmap/iD#3127]
Hello,

Sometimes I've got a problem on Chrome: way and area are unselectable, i can select poi.
Then I'm force to use shift+clic to select so it's really long.
I don't want to install Firefox on this computer. Sometimes it happens on my other one , and Firefox solves this issue.

Can you, please, try to fix this?

**Comment by homersimpsons** [comment]
It can be a duplicate I think. But I can select node. But I can select way and polygon only if I use shift. This happen on Windows 7 and windows 10, and restarting chrome doesn't help, then maybe it's not related to iD.

**Comment by bhousel** [comment]
Yes it does sound like a duplicate of that issue.

> restarting chrome doesn't help

Can you tell me what happens when you restart Chrome?   Do you close the iD tab completely and open a new one?  Do you restore your history?

**Comment by homersimpsons** [comment]
I'm always in private navigation, I close the window (all tab) and restart chrome. Will try to get java log when it will happen again. 
I often have just chrome (With gostery) and BOINC open.

**Comment by bhousel** [comment]
> I'm always in private navigation, I close the window (all tab) and restart chrome.

Ok, but when you restart Chrome, you can select again?  Or it remains broken?

**Comment by homersimpsons** [comment]
No I can't can select node, but no way/polygon) , that's why, maybe, it's not iD related... Will try to take care the next time, because when it happen I switch to Firefox.

**Comment by homersimpsons** [comment]
Got error currently, console shows :
"failed to load ressource: net::ERR_BLOCKED_BY_CLIENT"
and the link : "http://piwik.openstreetmap.org/piwik.js"
I let my computer 3 hours without editing, and disconnected from Internet, before this happen.

**Comment by homersimpsons** [comment]
if I refresh, I got : 
"GET http://piwik.openstreetmap.org/piwik.js net::ERR_BLOCKED_BY_CLIENTsend @ application-9c16b9d….js:26pe.extend.ajax @ application-9c16b9d….js:26(anonymous function) @ application-9c16b9d….js:216c @ application-9c16b9d….js:24d.fireWith @ application-9c16b9d….js:24pe.extend.ready @ application-9c16b9d….js:24s @ application-9c16b9d….js:14"
and link
"http://www.openstreetmap.org/assets/application-9c16b9dad23a368b98e54d15c3b6833d28bc731d7ccc544302bb4db3dc7c5930.js:26"

**Comment by bhousel** [comment]
Hmm - I think the piwik error is likely not causing the issue, but just in case, can you try editing with https://openstreetmap.us/iD/master/  

It would be interesting to know if you still get an error on there, which would be better because then it would be in unminified code.

**Comment by homersimpsons** [comment]
On that one I goth this error:
"GET https://dnn506yrbagrg.cloudfront.net/pages/scripts/0013/6714.js?406702 net::ERR_BLOCKED_BY_CLIENT"
and link:
"(index):36"
But currently I can select ways and polygons.

Maybe piwik is disabled by ghostery

**Comment by homersimpsons** [comment]
In fact piwik is always blocked by gostery...

**Comment by pnorman** [comment]
piwik is likely unrelated since #2151 is hit without blocking piwik

**Comment by pnorman** [comment]
That being said, it's best to duplicate bugs without any extensions.

**Comment by homersimpsons** [comment]
Have you look :
http://mourner.github.io/simplify-js/

In a first version, linked dot and poi dot should not be removed. In a second version, linked dot should be removed only if the 3 consecutive lot comes from the both same area

**Comment by bhousel** [comment]
@homersimpsons did you mean to put this on the "Simplify Way" issue #3116?  
(simplify-js is also Douglas-Peucker)

**Comment by homersimpsons** [comment]
Yes, sorry I put this in wrong ticket. In their demo, with high quality it looks a good simplification.

**Comment by Stalfur** [comment]
Since last update this problem (which is duplicate of older issue) is more prevalent than before. Making iD nearly impossible to use for anything beyond drawing up things - editing them is a PITA.

**Comment by bhousel** [comment]
closing here, lets keep discussion on #2151
